.github/workflows/arm-binary.yml

```diff
@@ -0,0 +1,70 @@
+name: Compile Binary for ARM
+
+on:
+  push:
+    branches:
+      - 'staging'
+      - 'trying'
+      - 'master'
+      - 'dev'
+  pull_request:
```

Comment:
Why can't this workflow just be another _job_ in the CI workflow? You are duplicating these triggers here.

Reply:
It can. I just want to keep it clean and not extend the mandatory checks with this added compilation.
I'll add it as a new `build (armv7)` job.

Reply:
I agree with you but if I want to include properly there is a cascade of changes (IMHO) as would replace the matrix from os based to target based.

I'll leave it like that at this stage.

Reply:
> I agree with you but if I want to include properly there is a cascade of changes (IMHO) as would replace the matrix from os based to target based.
> 
> I'll leave it like that at this stage.

That is even one step further from what ai envisioned which was just another job in the same workflow. That shouldn't affect the matrix :)
I am okay with leaving it as is though!

Reply:
As discussed, I'll try to do it properly with the release first.